### ISSUE TITLE
Use example for gtk::Dialog should contain a Rust example rather than a C example

### ISSUE THREAD

**Issue by marmistrz** [gtk-rs/gtk#1016]
https://gtk-rs.org/docs/gtk/struct.Dialog.html currently contains a C example.

**Comment by sdroege** [comment]
The docs are autogenerated from the C documentation, which unfortunately contains C examples. I don't think we can solve that in any nice way currently, @GuillaumeGomez ?

See https://github.com/gtk-rs/examples/ for some examples, some of which also use a dialog.

**Comment by GuillaumeGomez** [comment]
No, it's a discussion I had with people in GNOME and the outcome is: it's complicated.

**Comment by bilelmoussaoui** [comment]
I have heard that gtk is switching it's doc to markdown, maybe if they specify the language there we could have a separate rust docs with just the code snippets somehow?
